0.78 changelog (#31174)
microsoft/PowerToys: README.md

Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.

### 0.78 - January 2024 Update

In this release, we focused on stability and improvements.

**Highlights**

>  - New localization languages released: Arabic (Saudi Arabia), Hebrew, Persian and Ukrainian. We are going to assume we have some bugs. We want to fix them and are open for community help here too.

Reviewer: ```suggestion
 - New languages added: Arabic (Saudi Arabia), Hebrew, Persian and Ukrainian. We are going to assume we have some bugs. We want to identify & fix them and are open for community help.
```

>  - Many dependency updates, aiming for security and stability.

Reviewer: ```suggestion
 - Many dependencies updated, aiming for security and stability.
```

 - Fixed commonly reported PowerToys Run startup crashes after an upgrade.
 - New settings and GPO policies to help control behavior after an upgrade. Thanks [@htcfreek](https://github.com/htcfreek)!

### General

 - Added Arabic (Saudi Arabia) translation.
 - Added Hebrew translation.
 - Added Persian translation.
 - Added Ukrainian translation.
 - Improved the file watcher used across many utilities to consume less resources. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!

### AlwaysOnTop

 - Fixed an invisible border issue when the border color was set to the black color.
 - Added the AlwayOnTop icon to the base application executable. Thanks [@ckirby19](https://github.com/ckirby19)!

### Command Not Found

>  - Signed the PowerShell scripts used by Command Not Found installation process.

Reviewer: ```suggestion
 - Signed the PowerShell scripts used by the Command Not Found installation process.
```

### File Explorer add-ons

>  - Fixed an issue causing SVG Thumbnail generation to hang when trying to preview SVG files at the same time.

Reviewer: ```suggestion
 - Fixed an issue that caused SVG Thumbnail generation to hang when trying to preview multiple SVG files at the same time.
```

Author: This is not correct. You can't preview multiple SVG files at the same time in the preview pane.

Reviewer: Oh now I get it. Yeah then it is correct.

### File Locksmith

 - Improved the context menu entry caption. Thanks [@niels9001](https://github.com/niels9001)!

### Find My Mouse

>  - Added more settings to allow tuning shake detection when activating through mouse shake.

Reviewer: ```suggestion
 - Added more settings to tune shake detection when activating through mouse shake.
```

### Peek

 - Added a missing tooltip for the file size. Thanks [@HydroH](https://github.com/HydroH)!

### PowerRename

 - Improved and added localization to the context menu entry caption. Thanks [@niels9001](https://github.com/niels9001)!

### PowerToys Run

 - Removed references to unused settings from the code, which were causing crashes on some machines. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 - Fixed an issue causing a scrollbar to be out of view. Thanks [@niels9001](https://github.com/niels9001)!
 - Added logic to try and detect running games to full screen detection. Thanks [@anaisbetts](https://github.com/anaisbetts)!
 - Added support for converting negative values in the Unit Converter plugin. Thanks [@Dub1shu](https://github.com/Dub1shu)!
 - Fixed stale results in the Visual Studio Code Workspaces plugin by checking if files still exist. Thanks [@anderspk](https://github.com/anderspk)!
 - Fixed an activation crash that occurred after 0.77 on some configurations.
 - Fixed a startup crash that occurred when saving the new version of settings after an upgrade.

>  - Allow calculating bigger hexadecimal numbers in the Calculator plugin.
>  - Apply the "max results to show before scrolling" setting to the initial plugin hint listing as well.

Reviewer: ```suggestion
 - You can now calculate bigger hexadecimal numbers in the Calculator plugin.
 - The "max results to show before scrolling" setting can now also be applied to the initial plugin hint listing.
```

### Quick Accent

>  - Added the Ellipses character to all languages. Thanks [@HydroH](https://github.com/HydroH)!

Reviewer: ```suggestion
 - Added the ellipses character to all languages. Thanks [@HydroH](https://github.com/HydroH)!
```

>  - Added an option to try and not activate when playing a game. Thanks [@HydroH](https://github.com/HydroH)!

Reviewer: ```suggestion
 - Added an option to not activate when playing a game. Thanks [@HydroH](https://github.com/HydroH)!
```

 - Added the E with breve and pilcrow characters to all languages. Thanks [@PesBandi](https://github.com/PesBandi)!

### Settings

 - Removed the Command Not Found listing from the Settings dashboard and flyout, since it can't really be enabled or disabled from there.
 - Added a settings and GPO rule to disable opening the What's New OOBE page after an update. Thanks [@htcfreek](https://github.com/htcfreek)!
 - Added a settings and GPO rule to disable toast notifications about new updates being available. Thanks [@htcfreek](https://github.com/htcfreek)!
 - Fixed an issue causing the Settings window to not be brought to the foreground after activating through the system tray icon.
 - Standardized accent brush and corner radius on the dashboard page.
 - Improved UI and messages for GPO locked settings. Thanks [@htcfreek](https://github.com/htcfreek)!

>  - Fixed an issue causing the OOBE window to maximize and hide the system task bar.

Reviewer: ```suggestion
 - Fixed an issue causing the OOBE window to maximize and hide the system taskbar.
```

Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.

### 0.78 - January 2024 Update

In this release, we focused on stability and improvements.

**Highlights**

 - New languages added: Arabic (Saudi Arabia), Hebrew, Persian and Ukrainian. We are going to assume we have some bugs. We want to identify & fix them and are open for community help.
 - Many dependencies updated, aiming for security and stability.
 - Fixed commonly reported PowerToys Run startup crashes after an upgrade.

>  - New settings and GPO policies to help control behavior after an upgrade. Thanks [@htcfreek](https://github.com/htcfreek)!

Reviewer 2: ```suggestion
 - New settings and GPO policies to help control the update behavior. Thanks [@htcfreek](https://github.com/htcfreek)!
```
